vendor/vendor.json

```diff
@@ -2,12 +2,80 @@
 	"comment": "",
 	"ignore": "test",
 	"package": [
+		{
+			"checksumSHA1": "QTzx12uuB3JeMMvBE8EJH+4PTsI=",
+			"path": "github.com/cloudfoundry/jibber_jabber",
+			"revision": "bcc4c8345a21301bf47c032ff42dd1aae2fe3027",
+			"revisionTime": "2015-11-20T18:32:58Z"
+		},
 		{
 			"checksumSHA1": "CSPbwbyzqA6sfORicn4HFtIhF/c=",
 			"path": "github.com/davecgh/go-spew/spew",
 			"revision": "d8f796af33cc11cb798c1aaeb27a4ebc5099927d",
 			"revisionTime": "2018-08-30T19:11:22Z"
 		},
+		{
```

Comment:
How did we bring on all these libs? lots of ones related to color console output as well as YAML parsing?  We should not be using these in the go-client (perhaps in the CLI).  Did we accidentally pick up all godeps used by the combined CLI (plugin)?

Reply:
Prior to this PR (which does not seem to use any new external libs.) we only had vendor.json with these libs:
- github.com/davecgh/go-spew/spew
- github.com/pmezard/go-difflib/difflib
- github.com/stretchr/testify/assert